### PR TITLE
More testing

### PR DESCRIPTION
**Pull request by bbonf**

### CODE REVIEW

File: spec/browser-info.spec.mjs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    it('should not be empty', () => {
+        expect(Object.keys(info).length).toBeGreaterThan(0);
+    });
+
+    it('should be correct for headless chrome', () => {
+        expect(info.browser).toContain('Chrome Headless');
+        expect(info.isMobile).toBeFalse();
+        expect(info.isTablet).toBeFalse();
+        expect(info.isTouchCapable).toBeFalse();
+    });
```
Comment (maartenuni) [inline spec/browser-info.spec.mjs:15]: This one might be a bit of a pain. I'd like to run the test with
```bash
npx jasmine-browser-runner
```
and then navigate to localhost:8888 in firefox or chrome
This is great for the CI i suppose, but perhaps less nice for running the test with a browser and its development tools.

Comment (bbonf) [reply spec/browser-info.spec.mjs:15]: I think the browser name part can go, it's not super important

**Comment by maartenuni** [comment]
I don't have a approve button anymore, feel free to merge it when your ready.

**Review by maartenuni — APPROVED** [review]

**Comment by bbonf** [comment]
I just wanted to remove the browser name first

**Pull request merged**